packages/backend/src/open_payments/payment/incoming/model.ts

```diff
@@ -245,7 +245,7 @@ export class IncomingPayment
     ilpStreamConnection: Connection
   }): OpenPaymentsIncomingPayment {
     return {
-      id: this.id,
+      id: this.url,
```

Comment:
OpenPayment API IncomingPayments return the full `url` instead of just the uuid. This reflects what is done in the IncomingPaymentRoutes, but standardizes it when converting between `IncomingPayment` model and what Open Payments expects

Reply:
Although I'm not sure how this plays with https://github.com/interledger/rafiki/issues/662, I'll have to take a look at how the `IncomingPaymentsJSON` type is used

Reply:
:+1: